### PR TITLE
port #17618 to the 2.1 branch (Disable timeout test)

### PR DESCRIPTION
**Pull request by sandreenko**
disable DevDiv_255294 in stress modes. (PR to master #17618, issue #17605) 

It is a test only change.

### CODE REVIEW

**Comment by sandreenko** [comment]
@RussKeldorph, the issue was labeled as 2.1, do we want to port this change?

**Comment by RussKeldorph** [comment]
Yes, you can merge it once CI is green.

File: tests/src/JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294.csproj [hunk 1]
```diff
@@ -23,6 +23,8 @@
   <PropertyGroup>
     <DebugType></DebugType>
     <Optimize>True</Optimize>
+	<!-- It hits timeout with R2R and JitStress=1, issue 16573 -->
+	<JitOptimizationSensitive>true</JitOptimizationSensitive>
```
Comment (RussKeldorph) [inline tests/src/JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294.csproj]: Whitespace looks wrong.

**Comment by sandreenko** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test

**Pull request merged**